src/main/java/seiki/data/TaskList.java

```diff
@@ -0,0 +1,93 @@
+package seiki.data;
+
+import java.util.ArrayList;
+
+import seiki.data.task.Task;
+
+/**
+ * Represents the entire task list.
+ * Contains the data of the task list.
+ */
+public class TaskList {
+    protected ArrayList<Task> taskList;
+    protected int taskCount;
+
+    /**
+     * Creates an empty task list.
+     */
+    public TaskList() {
+        this.taskList = new ArrayList<>();
+        this.taskCount = 0;
+    }
+
+    /**
+     * Constructs a task list with the given data.
+     * @param tasks
+     */
+    public TaskList(ArrayList<Task> tasks) {
+        this.taskList = tasks;
+        this.taskCount = tasks.size();
```

Comment:
Perhap 'this' keyword should be omitted?

Reply:
Oh! It slipped my mind. Thank you for spotting it!